content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Add a job to TestGrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+This document lays out the process of getting a CI job recorded as blocking or informing a release as well as the process for having job outputs exposed in TestGrid page. While all jobs that block or inform a release are necessarily exposed in TestGrid pages, it is not required that a job have influence on an OpenShift release for the job's output to be present in a TestGrid overview.
+To just understand how a job is added to TestGrid skip to [How do I add a job to TestGrid?](#how-do-i-add-a-job-to-testgrid)
+## What is TestGrid?
+TestGrid is an aggregator for job results across many runs, allowing us to display test results over time and thus giving an indication of current health, especially in the face of flaky tests. Note that TestGrid exposes job results only from jUnit.
+
+## How can CI Jobs impact the OpenShift release status?
+* Blocking jobs are those that ensure compliance to our conformance criteria. These are traditionally the core end-to-end test runs on our major platforms and upgrades from previous versions. 
+* Informing jobs are a broader suite that test the variety of enviroments and configurations our customers expect. 
+* Broken jobs are those that have a known, triaged failure that prevents their function for a sustained period of time (more than a week).
+
+## When does a job qualify as a release-informing job?
+Since release informing jobs get run every time we build a new payload for that release we need to ensure that the given job:
+1) is reliable enough to provide a useful signal when it fails
+2) covers an important function that determines whether the payload is viable or not, which is not covered by other informing or blocking jobs
+
+
+## How do I add a job to TestGrid?
+
+#### Adding a release gating job to TestGrid
+Release gating jobs are picked up automatically to be added to TestGrid. If the job has already been configured in the release controller nothing further needs to be done. For information on how to configure a job to be release gating refer to [How do I set a job to be release-gating?](#how-do-i-set-a-job-to-be-release-gating)
+**Note:** If a job in the release definition is an upgrade job it goes into the overall informing dashboard (because upgrades cross dashboards).
+
+#### Adding a non-release gating job to TestGrid
+If the job has an entry in [_allow-list.yaml](https://github.com/openshift/release/blob/master/core-services/testgrid-config-generator/_allow-list.yaml) the job will get added to TestGrid. This will override the default on the job (unless the job is blocking on the release controller and the annotation is informing) if it was a release gating job. Refer [/ci-tools/testgrid-config-generator](https://github.com/openshift/ci-tools/tree/master/cmd/testgrid-config-generator) for more details regarding how the testgrid-config-generator works.
```

Comment:
```suggestion
If the job has an entry in [_allow-list.yaml](https://github.com/openshift/release/blob/master/core-services/testgrid-config-generator/_allow-list.yaml) the job will get added to TestGrid. This will override the default on the job (unless the job is blocking on the release controller and the annotation is informing) if it was a release gating job. Refer to the README for [/ci-tools/testgrid-config-generator](https://github.com/openshift/ci-tools/tree/master/cmd/testgrid-config-generator) for more details regarding how the testgrid-config-generator works.
```

Reply:
or `s|/ci-tools/testgrid-config-generator|the TestGrid config generator tool|`